src/xenia/cpu/backend/x64/x64_sequences.cc

```diff
@@ -1774,10 +1901,30 @@ struct DIV_I32 : Sequence<DIV_I32, I<OPCODE_DIV, I32Op, I32Op, I32Op>> {
       assert_true(!i.src1.is_constant);
       e.mov(e.ecx, i.src2.constant());
       if (i.instr->flags & ARITHMETIC_UNSIGNED) {
-        e.mov(e.eax, i.src1);
-        // Zero upper bits.
-        e.xor_(e.edx, e.edx);
-        e.div(e.ecx);
+        auto div_shift_info = magicu<unsigned>(i.src2.value->constant.u32);
+		//todo: handle the add flag. 
```

Comment:
1. Tabs instead of spaces.
1. Don't leave this silently unhandled. Add an assert for this case where the implementation should go.